### PR TITLE
always hide button to close group of tabs

### PR DESCRIPTION
**Pull request by kecnry**
### Description

This pull request always hides the "x" button in the top right of a golden layout panel (which otherwise would close all viewers in that tab group), so that we can prevent closing specific viewers via css filters on the tab itself.  Otherwise, users could move a "protected viewer" tab with "x" disabled into a group, and then ultimately delete the viewer by closing the entire group.

### Change log entry

- [x] Is a change log needed? If yes, is it added to `CHANGES.rst`? If you want to avoid merge conflicts,
  list the proposed change log here for review and add to `CHANGES.rst` before merge. If no, maintainer
  should add a `no-changelog-entry-needed` label.

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [x] Are two approvals required? Branch protection rule does not check for the second approval. If a second approval is not necessary, please apply the `trivial` label.
- [ ] Do the proposed changes actually accomplish desired goals? Also manually run the affected example notebooks, if necessary.
- [ ] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Did the CI pass? If not, are the failures related?
- [x] Is a milestone set? Set this to bugfix milestone if this is a bug fix and needs to be released ASAP; otherwise, set this to the next major release milestone.
- [ ] After merge, any internal documentations need updating (e.g., JIRA, Innerspace)?

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -79,6 +79,8 @@ Bug Fixes
 
 - Fixes viewer toolbar items losing ability to bring up right-click menu. [#2605]
 
+- Disable closing tab-groups of viewers to prevent closing required viewers. [#2625]
+
```
Comment (pllim) [inline CHANGES.rst:83]:
```suggestion
```

Comment (kecnry) [reply CHANGES.rst:83]: I suppose that's true!  Any config that had the tabs enabled at all had this manually hidden already.

**Review by pllim — APPROVED** [review]
Let's get this in then. Thanks!

**Comment by pllim** [comment]
Hmm Kyle, auto backport failed. Do you want to manually backport or change the milestone?

**Pull request merged**